app/lib/generate_consent_documents.rb

```diff
@@ -0,0 +1,37 @@
+class GenerateConsentDocuments
+  def create_consent_docs
+    # GenerateConsentDocuments.new.create_consent_docs
+    intakes_without_consent_doc = []
+    successes = []
+    problems = []
+
+    Intake.all.each do |intake|
```

Comment:
It would be substantially more efficient to do this with some kind of `JOIN` query, but I suppose it doesn't matter much if we run this in the background and/or on a temporary `aptible ssh` worker.